### PR TITLE
Document process for adding new jupyterlab repos

### PR DESCRIPTION
**Pull request by saulshanabrook**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
 A repository for team interaction, syncing, and handling meeting notes across the JupyterLab ecosystem.
+
+
+## Process for adding new projects
+
+If you have an existing JupyterLab extension, or would like to create one, this is the process
+for doing so on the core `jupyterlab` organization:
+
+1. You post an issue on this repo describing your extension and linking to any existing code. You should describe who will maintain it and why it will be useful to the community.
+2.  A core maintainor reviews any existing code as a sanity check.
```
Comment (afshin) [inline README.md]: sp. (`maintainor` => `maintainer`)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+## Process for adding new projects
+
+If you have an existing JupyterLab extension, or would like to create one, this is the process
+for doing so on the core `jupyterlab` organization:
+
+1. You post an issue on this repo describing your extension and linking to any existing code. You should describe who will maintain it and why it will be useful to the community.
+2.  A core maintainer reviews any existing code as a sanity check.
+3.  Relicense any existing code to https://github.com/jupyterlab/team-compass/blob/master/LICENSE.
+4.  Give the community roughly a week to comment on the proposal.
+5. Pending any objections, create a new repository. Give any maintainers admin privilages on that repo and add them as maintainers on the relvent NPM packages.
```
Comment (afshin) [inline README.md]: sp. `privilages` => `privileges`

Comment (afshin) [reply README.md]: sp. `relvent` => `relevant`

**Comment by jasongrout** [comment]
I took a pass at this as well: https://github.com/jasongrout/team-compass/tree/adoption

Do you want me to make a PR to your branch, or just push directly to your branch so that my changes show up here in this PR?

**Comment by saulshanabrook** [comment]
@jasongrout feel free to push directly to my branch!

**Comment by saulshanabrook** [comment]
@jasongrout do you have further edits to this?

**Comment by saulshanabrook** [comment]
I am merging this for now. We can always iterate more.

**Pull request merged**